cluster-scope/base/core/namespaces/okd-team/okd-quotas.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ResourceQuota
+apiVersion: v1
+metadata:
+  name: large
```

Comment:
Please rename the resource to `okd-custom` (filename should be `resourcequota.yaml`)

Reply:
Sure!
thanks so much